doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -344,12 +355,40 @@ Server and client by default will run on localhost address. Run server:
    :width: 700
    :align: center
 
-Run client:
+Testing Zephyr Host Stack on QEMU:
+
+.. code-block::
+
+    # Attach Bluetooth Controller (with H4 is an example)
```

Comment:
It would be better to refer to already existing documentation sections for attaching bluetooth controllers.
https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-tools.html#running-on-qemu-and-native-posix